Sources/PackageDescription/Package.swift

```diff
@@ -185,6 +185,24 @@ public func ==(lhs: Package.Dependency, rhs: Package.Dependency) -> Bool {
 
 // MARK: Package Dumping
 
+struct Errors {
+    /// Storage to hold the errors.
+    private var errors = [String]()
+
+    /// Adds error to global error array which will be serialized and dumped in TOML at exit.
+    mutating func add(_ str: String) {
+        errors += ["\"\(str)\""]
```

Comment:
Yeah if a string is coming here means it has to be a _valid_ swift string. Actually this quoting really belongs in the method `toTOML` but it doesn't really matter at this point of time because there is no other use case. I put it here because the compiler was erroring out with "expression is too complex to resolve" (which I could resolve by breaking into multiple statements but I _think_ its fine for now). What do you think?